scripts/workflows/wf.yml

```diff
@@ -0,0 +1,86 @@
+options:
+  env:
+    DEVICE: /dev/nvme0n1
+
+steps:
+- id: build-img
+  uses: docker://docker:19.03.10
+  dir: /workspace/scripts/perf/docker
+  args: [build, --tag=spdk, .]
+
+- id: prepare
+  uses: docker://spdk
+  skip_pull: true
+  runs: [bash, -ex, -c]
+  args:
+  - |
+    nvme format /dev/nvme0 --ses=2 -n=1
+    echo 512 > /proc/sys/vm/nr_hugepages
+    sysctl -w vm.nr_hugepages=512
```

Comment:
do you need both, `echo` and `sysctl`? I was under the impression that only one of them is needed since they accomplish the same

Reply:
sorry again, actually I ended up not needing those lines, it was not absolutely required to set hugepages to run the tests with the spoke engine

Reply:
no worries 😄